doc/releases.md

```diff
@@ -0,0 +1,51 @@
+# Releases
+
+https://github.com/badges/shields is a monorepo and hosts the Shields frontend and server code as well as the [badge-maker][npm package] NPM library (and the [badge design specification](https://github.com/badges/shields/tree/master/spec)). The packaging and release processes for these respective items is described below.
+
+## badge-maker package
+
+We follow [Semantic Versioning](https://semver.org/) for the badge-maker package, and publish git Tags using the form `X.Y.Z`, and all such tags of this form are badge-maker releases (e.g. [3.3.0](https://github.com/badges/shields/releases/tag/3.3.0)
```

Comment:
Minor: Missing close bracket here. You've closed the markdown link, but not the  `(e.g. ...`